editor/plugins/script_text_editor.cpp

```diff
@@ -1186,7 +1186,7 @@ void ScriptTextEditor::_edit_option(int p_op) {
 				if (expression.parse(line) == OK) {
 					Variant result = expression.execute(Array(), Variant(), false);
```

Comment:
You should remove this variable then.

Reply:
But it's still used.

Reply:
Ah my bad, I misread the remaining of the line. Everything looks good then. :)